src/index.js

```diff
@@ -21,16 +36,61 @@ const App = (appProps) => {
     );
   }
 
+  const goToNew = () => {
+    history.push(`${location.pathname}/create`);
+  };
+
+  const shortcuts = [
+    {
+      name: 'new',
+      handler: goToNew,
+    },
+    {
+      name: 'openShortcutModal',
+      handler: () => setIsShortcutsModalOpen(true),
+    }
+  ];
+
   return (
-    <Suspense fallback={null}>
-      <Switch>
-        <Route component={WidgetCreateRoute} path={`${path}/:dashName/create`} />
-        <Route component={WidgetCreateRoute} path={`${path}/:dashName/:widgetId/edit`} />
-        <Route component={DashboardOrderRoute} path={`${path}/:dashName/editOrder`} />
-        <Route component={DashboardRoute} path={`${path}/:dashName`} />
-        <Route component={DashboardsRoute} path={path} />
-      </Switch>
-    </Suspense>
+    <>
+      <CommandList commands={defaultKeyboardShortcuts}>
+        <HasCommand
+          commands={shortcuts}
+          isWithinScope={checkScope}
+          scope={document.body}
+        >
+          <AppContextMenu>
+            {(_handleToggle) => (
+              <NavList>
+                <NavListSection>
+                  <NavListItem
+                    id="keyboard-shortcuts-item"
+                    onClick={() => { setIsShortcutsModalOpen(true); }}
+                  >
+                    <FormattedMessage id="ui-dashboard.appMenu.keyboardShortcuts" />
+                  </NavListItem>
+                </NavListSection>
+              </NavList>
+        )}
+          </AppContextMenu>
+          <Suspense fallback={null}>
```

Comment:
@CalamityC Now that we have moved away from using `React.lazy`, we should move the usage of `Suspense` out as well, as it serves no purpose here. The routes imports are not dynmamic anymore.

Reply:
It's giving me another error when I remove the `<Suspense fallback={null}>`, see my comments above

> * keep Suspense, removal is causing another error